services/utils/__init__.py

```diff
@@ -18,15 +18,6 @@
 logging.basicConfig(level=log_level)
 
 
-async def read_body(request_content):
-    byte_array = bytearray()
-    while not request_content.at_eof():
-        data = await request_content.read(4)
-        byte_array.extend(data)
-
-    return json.loads(byte_array.decode("utf-8"))
-
-
```

Comment:
Were there some issues encountered with using `await response.json()` or why was this previously necessary? Concerned about some possible edge cases that motivated the previous implementation

Reply:
I will check with Ferras for the why of the previous implementation but I felt it would be better to use the standard implementation instead of doing our own.